signer/src/transaction_signer.rs

```diff
@@ -169,7 +169,9 @@ where
 
                 // If we've observed a new block, we need to handle any new requests.
                 if new_block_observed {
-                    self.handle_new_requests().await?;
+                    if let Err(error) = self.handle_new_requests().await {
+                        tracing::warn!(%error, "error handling new requests; skipping this round");
+                    }
```

Comment:
we are not skipping the round here (not sure if we should though)

Reply:
i guess i meant more that we're skipping "handling new requests" for this round